src/shared/times-option.js

```diff
@@ -0,0 +1,91 @@
+/// <reference path="./types.js" />
+
+/**
+ * @module shared
+ */
+
+/**
+ * @type AriaATCIShared.timesOption
+ */
+export const timesOption = {
+  afterNav: 1000,
+  afterKeys: 5000,
+  testSetup: 1000,
+  modeSwitch: 750,
+  docReady: 2000,
+};
+
+/**
+ * @type AriaATCIShared.timesOption
+ */
+const timesDefaults = { ...timesOption };
+
+/**
+ * Convert from 'afterNav' to 'time-after-nav'.
+ * @param {keyof AriaATCIShared.timesOption} optionName
+ * @returns string
+ */
+function makeSnakeCasedOption(optionName) {
+  const snakeCased = optionName.replace(/[A-Z]/g, cap => '-' + cap.toLowerCase());
+  const optionText = `time-${snakeCased}`;
+  return optionText;
+}
```

Comment:
I kinda have a thing for grep-able source code. I'd much rather maintain a bit of redundancy (e.g. an argument list like `"afterNav", "time-after-nav"`) than an abstraction like this.

Reply:
pushed this change in